config/postcss.config.js

```diff
@@ -24,7 +24,8 @@ module.exports = {
     sourcesContent: true,
     plugins: [
         postcssImport(),
-        postcssAddFallback({importFrom: 'dist/root.css'}), //replace with theme-designer variables
+        postcssAddFallback({importFrom: 'dist/root.css'}),
+        postcssAddFallback({importFrom: 'node_modules/@sap-theming/theming-base-content/content/Base/baseLib/sap_fiori_3/css_variables.css'}),
```

Comment:
The `importFrom` does not accept an array of files so we just needed to call this plugin twice.